packages/sdk/CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
+* fetch and load masp params ([#1136](https://github.com/anoma/namada-interface/issues/1136)) ([5df8577](https://github.com/anoma/namada-interface/commit/5df8577cf0627247a7aeb5fa4de5e3970049b024))
+* hook up interface to SDK package, extension signing ([1e2ad8e](https://github.com/anoma/namada-interface/commit/1e2ad8e4ff3c64451e94d36ef9559180fbcd27c5))
+* hook up tx hash to build Tx, add to SDK ([#753](https://github.com/anoma/namada-interface/issues/753)) ([35909e7](https://github.com/anoma/namada-interface/commit/35909e7a2cdba35fea0f0af46b8628de240d420e))
+* **interface:** fixing namadillo dev serve ([c66af51](https://github.com/anoma/namada-interface/commit/c66af51f59c6b4d611e0c69911e73e6605678191))
+* move sdk package into [@namada](https://github.com/namada) org ([#1230](https://github.com/anoma/namada-interface/issues/1230)) ([0376020](https://github.com/anoma/namada-interface/commit/0376020411a6b123376a39bce4240bb7468858ae))
+* post-release bumps, changelogs ([#1364](https://github.com/anoma/namada-interface/issues/1364)) ([a2f3f5c](https://github.com/anoma/namada-interface/commit/a2f3f5cd54ff4bf905b385dc58a9c5d44c2c4ba4))
+* sdk api improvements ([#695](https://github.com/anoma/namada-interface/issues/695)) ([6eee06c](https://github.com/anoma/namada-interface/commit/6eee06cb2b40eac8bd2eebc399fc87698654aaa1))
+* sdk js publish ([#665](https://github.com/anoma/namada-interface/issues/665)) ([ec3dec8](https://github.com/anoma/namada-interface/commit/ec3dec8070219f29ccf95e8a50c880da3f032566))
+* sdk version bump, changelog ([#1175](https://github.com/anoma/namada-interface/issues/1175)) ([f70a6a4](https://github.com/anoma/namada-interface/commit/f70a6a4c1e1d8ddef3e2985ec16295c666f4c95b))
+* shield nam ([#1165](https://github.com/anoma/namada-interface/issues/1165)) ([e6b5530](https://github.com/anoma/namada-interface/commit/e6b55307c77312a3bdde192ec721d5e84883d4ba))
+* support claim rewards tx in SDK ([#932](https://github.com/anoma/namada-interface/issues/932)) ([59d0ea9](https://github.com/anoma/namada-interface/commit/59d0ea9659658c23c804324d46594783ed695a2e)), closes [#679](https://github.com/anoma/namada-interface/issues/679)
+* support decoding ibc tx details in sdk ([#1174](https://github.com/anoma/namada-interface/issues/1174)) ([1bd083b](https://github.com/anoma/namada-interface/commit/1bd083ba5883a3f722a6838b9694223afeae71a1))
+* update unbonding period calculation ([#1034](https://github.com/anoma/namada-interface/issues/1034)) ([2e65ae8](https://github.com/anoma/namada-interface/commit/2e65ae8ae3d2430b6268603785c30016d2df77a6))
+* update zip32 path ([#621](https://github.com/anoma/namada-interface/issues/621)) ([c4e5413](https://github.com/anoma/namada-interface/commit/c4e54131064b1d7df3704a0f815cd041dc551740))
```

Comment:
Not sure why these ^^^ got picked up this time, maybe there weren't included in the previous one? Not a big deal, something to look into. The Bug fixes below looks correct!